specification/common-types/resource-management/v3/managedidentity.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
```

Comment:
Do you have any plan in mind to refresh SDK? I think it would take a very long time. Is it possible we split this PR into several PRs containing only one file, or only one version? I hope we do validate/refresh one by one.

Reply:
I feel like we cannot do the validation at SDK side. I'll explain this to you in our sync.
![image](https://github.com/user-attachments/assets/a9c12de7-b68e-405c-91bd-93f3f9132744)
Considering this, I think the only change we could accept is description change.

Reply:
ok, let's discuss this  :)

Reply:
I agree this should be involved in refreshing the SDK, and also it needs to be *validated* against the SDK, since some of the changes might be visible in code generation side effects. What are the barriers to doing this? 

/cc @markcowl @rkmanda @mikeharder

Reply:
We had some offline conversations around this topic and the problems around the validation but after some conversations this can be validated against the SDK. @pshao25 can we resolve this conversation? or there are still problems to perform the validaiton?

Reply:
Yes, I already have a script to validate against all the SDKs. I'll share it across languages.